docs/creating-studies.md

```diff
@@ -165,6 +165,57 @@ styling.
   your study from scratch due to a change in your counts output. If this is not
   set, the version will implicitly be set to zero.
 
+## Testing studies
+
+If you have a Cumulus database in Athena already,
+you can easily point at that during study development.
+
+But it may also be faster or easier to work with local files,
+where you can add edge cases.
+Cumulus Library has an optional database backend driven by local ndjson just for that!
+
+### Set up your ndjson
+
+You can grab fake
+[Synthea data](https://github.com/smart-on-fhir/sample-bulk-fhir-datasets),
+or use the result of actual bulk-export results from your EHR.
+
+Once you have that, run Cumulus ETL on your source ndjson with the
```

Comment:
nit: this is the first time in this doc we mention the ETL, so we should probably link to the ETL section of the docsite here.